Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:18.04
+
+# build essentials, python3, qemu and other deps
+RUN apt-get update && apt-get install -y software-properties-common
+RUN apt-add-repository -y universe
+RUN apt-get install -y \
+	build-essential \
+	gcc-multilib \
+	libtool \
+	automake \
+	autoconf \
+	bison \
+	git \
+    gcc \
+	debootstrap \
+ 	debian-archive-keyring \
+        libtool-bin
+
+# for QEMU compiling
+RUN cp /etc/apt/sources.list /etc/apt/sources.list~
+RUN sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list
+RUN apt-get update 
+RUN DEBIAN_FRONTEND=noninteractive apt-get build-dep -y \
+	qemu
+RUN apt-get install -y \
+	python3 \
+	python3-pip 
+
+# PHUZZER & DEPS
+RUN pip3 install git+https://github.com/shellphish/shellphish-afl
```

Comment:
We may want to consider installing this as a wheel. An old version is on pypi, but we also have a wheel in the wheels repo.

Reply:
Changed to a wheel.